Code/Legacy/CrySystem/Log.cpp

```diff
@@ -201,28 +197,16 @@ bool CLog::OpenLogFile(const char* filename, int mode)
         return false;
     }
 
-    // it is assumed that @log@ points at the appropriate place (so for apple, to the user profile dir)
-    AZ::IO::FileIOBase* fileSystem = AZ::IO::FileIOBase::GetDirectInstance();
-    if (AZ::IO::FixedMaxPath logFilePath; fileSystem->ReplaceAlias(logFilePath, filename))
-    {
-        logFilePath = logFilePath.LexicallyNormal();
-        m_logFileHandle.Open(logFilePath.c_str(), mode);
-    }
+    bool opened = m_logFileHandle.Open(filename, mode);
```

Comment:
We don't need to resolve aliases anymore here?

Reply:
No. When using `FileIOStream` it will resolve the aliases internally using the current FileIO System setup. If it's running normally it will be LocalFileIO and will replace the aliases normally, if it's RemoteFileIO it will sent messages remotely (this is what VFS sets up when it's used).

Reply:
The one danger though is if the FileIOBase instance isn't available, then this command will assert.

Reply:
I haven't seen any asserts triggered in my tests, but I will add a check that if FileIOBase is not available to not do a log to file.

Reply:
done